src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -3161,77 +3389,49 @@ impl Coordinator {
         };
 
         match action {
+            // TODO: Dropping a subsource via `ALTER SOURCE` only removes the
+            // references to the object
```

Comment:
What would it ideally also do?

Reply:
Ah, this is actually done in this PR and should have been removed.